src/saga/mergeUtils.js

```diff
@@ -1,21 +1,40 @@
 import { conflictType } from "../constants";
 
+function checkEmpty(row) {
+    const filteredRow = row.filter(element => element != "");
+    return filteredRow.length == 0 
+}
+
+function handleOriginUndefinedRow(a, b, possibleConflictType, rowIndex, colIndex) {
```

Comment:
Remove the colIndex as input to this function (and in the call) - and just pass in null in the call to createOriginUndefinedConflict below.